css/styles.css

```
@@ -47,3 +58,118 @@ body {
   height: 26px;
   fill: #fff;
 }
+
+/* CREATE NEW TODO */
+.new-todo {
+  margin-top: 48px;
+  margin-bottom: 24px;
+  padding: 20px 24px;
+  border-radius: 5px;
+  background: #fff;
+  box-shadow: 0px 35px 50px -15px rgba(194, 195, 214, 0.5);
+
+  display: flex;
+  gap: 24px;
+}
+
+.new-todo__circle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.new-todo__input {
+  border: none;
+  flex: 1;
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 18px;
+  font-weight: 400;
+  letter-spacing: -0.25px;
+  color: #393a4b;
+}
+
+.new-todo__input::placeholder {
+  color: #9495a5;
+}
+
+/* TODO LIST */
+.todo-list {
+  border-radius: 5px;
+  background: #fff;
+  box-shadow: 0px 35px 50px -15px rgba(194, 195, 214, 0.5);
+
+  display: flex;
+  flex-direction: column;
+}
+
+.checkbox__input + span {
+  width: 24px;
+  height: 24px;
+  border-radius: 50%;
+  border: solid 1px #e3e4f1;
+
+  display: inline-flex;
+}
+
+.checkbox__input:checked + span {
+  background-image: url(../assets/checked.png);
+  background-size: cover;
+}
+
+.checkbox__input {
+  display: none;
+}
+
+.todo-list__item--checked {
+  color: #d1d2da;
+  text-decoration: line-through;
+}
+
+.todo-list__item {
+  color: #494c6b;
+}
+
+.todo-list__item--checked,
+.todo-list__item {

```

no HTML onde tem o  `todo-list__item--checked`, você pode adicionar a `class` referente ao estilo base `todo-list__item`, ficando `class="todo-list__item todo-list__item--checked"`, feito isso tu não precisa adicionar os estilos base no `todo-list__item--checked`, removendo o seletor dele.

```suggestion
.todo-list__item {

```